src/App.tsx

```diff
@@ -17,18 +37,19 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter onFilter={setAppTodo} />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length
```

Comment:
I believe that we need to hide the loader inside the condition in case we dont have our information loaded , but the table can sit outside, basically if it has no information we will be able to see the table header empty. and the loader on top